packages/app/src/main/config.ts

```diff
@@ -2,6 +2,7 @@ const config = {
   applicationName: "Coh2 Game Stats",
   checkCurrentVersionURL: "https://coh2stats.com/electron-app-version.json",
   checkCurrentVersionLocalDevURL: "http://localhost:3000/electron-app-version.json",
+  mixpanelProjectToken: "40dec2eae51580f28e7ee646014b95cc",
```

Comment:
btw this is big no no, someone could take out key and mess up our analytics - this should be as env variable and stored in github secrets - automatically added when there is a build ... but I don't know right now how to setup the env variables for electron